src/items/bed.cpp

```diff
@@ -122,6 +117,23 @@ bool BedItem::canUse(std::shared_ptr<Player> player) {
 	return true;
 }
 
+bool BedItem::isBedComplete(std::shared_ptr<BedItem> nextBedItem) {
+	const ItemType &it = Item::items[id];
+
+	if (nextBedItem == nullptr) {
+		return false;
+	}
+
+	auto partName = it.name;
+	auto nextPartname = nextBedItem->getName();
+	auto firstPart = keepFirstWordOnly(partName);
+	auto nextPartOf = keepFirstWordOnly(nextPartname);
+
+	g_logger().debug("First bed part id {} name {}, second part id {} name {}", it.id, firstPart, nextBedItem->getID(), nextPartOf);
```

Comment:
I think you can remove parameters that was be used just one time
```suggestion
	auto firstPart = keepFirstWordOnly(it.name);
	auto nextPartOf = keepFirstWordOnly(nextBedItem->getName());
	g_logger().debug("First bed part id {} name {}, second part id {} name {}", it.id, firstPart, nextBedItem->getID(), nextPartOf);
```

Reply:
Agreed!

Reply:
I was trying but it shows the error `qualifiers dropped in binding reference of type "std::string &" to initializer of type "const std::string"`, so it is not possible.

Reply:
"I suspected from the beginning" - COLORADO, Chapolin